docs/exchange-operators/rosetta/build-from-sources.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Building from sources
+description: Build Mina’s implementation of Rosetta API from sources.
+keywords:
+  - historical data
+  - blockchain integration
+  - explorer
+  - wallet
+  - archive node
+  - graphQL
+  - rosetta
+  - nix
+---
+
+
+# Building and running Rosetta from source code
+
+In the [previous article](run-with-docker) we described the Docker cointainer, that runs mina daemon, mina-archive and Rosetta API. 
```

Comment:
```suggestion
In [Running with Docker](run-with-docker), you learned that the Docker container runs the mina daemon, mina-archive, and Rosetta API. 
```

Reply:
thanks for this PR, here's some resources you can rely on https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide 

- instead of "we" third person, docs style is to use second person "you" pronoun 
- instead of "previous article", use descriptive link text
- we love the serial comma!